.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: eslint
         types: [file]
         files: \.[jt]s$|vue$ # *.js, *.ts and vue
-        exclude: 'mocharc.js$'
+        exclude: '(\/|^)((.*\.ignored\.([jt]sx?|vue))|(\..*\.([jt]sx?|vue)))$'
```

Comment:
the regex is quite scary, but unfortunately pre-commit doesn't give us any better ways to filter out files